package.json

```diff
@@ -34,6 +34,7 @@
     "@theforeman/stories": "^8.7.0",
     "@theforeman/test": "^8.7.0",
     "@theforeman/vendor-dev": "^8.7.0",
+    "@types/jest": "<27.0.0",
```

Comment:
Shouldn't this be `@jest/types`? ;)

Reply:
I think I got this one right:
![types-jest](https://user-images.githubusercontent.com/2664090/134638085-92e079b0-7cb4-4710-ac3b-6b5f462ec1f2.png)

Reply:
huh, okay, the pr title was `jest/types`, so I thought that's correct ;)